README.md

```diff
@@ -101,53 +101,54 @@ determining that location is as follows:
-| shared\_vpc | The ID of the host project which hosts the shared VPC | string | `""` | no |
-| shared\_vpc\_subnets | List of subnets fully qualified subnet IDs (ie. projects/$project_id/regions/$region/subnetworks/$subnet_id) | list(string) | `<list>` | no |
-| usage\_bucket\_name | Name of a GCS bucket to store GCE usage reports in (optional) | string | `""` | no |
-| usage\_bucket\_prefix | Prefix in the GCS bucket to store GCE usage reports in (optional) | string | `""` | no |
+| lien | Add a lien on the project to prevent accidental deletion | string | `false` | no |
+| name | The name for the project | string | - | yes |
+| org_id | The organization ID. | string | - | yes |
+| project_id | If provided, the project uses the given project ID. Mutually exclusive with random_project_id being true. | string | `` | no |
+| random_project_id | Enables project random id generation. Mutually exclusive with project_id being non-empty. | string | `false` | no |
+| sa_role | A role to give the default Service Account for the project (defaults to none) | string | `` | no |
+| shared_vpc | The ID of the host project which hosts the shared VPC | string | `` | no |
+| shared_vpc_subnets | List of subnets fully qualified subnet IDs (ie. projects/$project_id/regions/$region/subnetworks/$subnet_id) | list(string) | `<list>` | no |
+| usage_bucket_name | Name of a GCS bucket to store GCE usage reports in (optional) | string | `` | no |
+| usage_bucket_prefix | Prefix in the GCS bucket to store GCE usage reports in (optional) | string | `` | no |
 
 ## Outputs
 
 | Name | Description |
 |------|-------------|
 | domain | The organization's domain |
-| group\_email | The email of the GSuite group with group_name |
-| project\_bucket\_self\_link | Project's bucket selfLink |
-| project\_bucket\_url | Project's bucket url |
-| project\_id |  |
-| project\_name |  |
-| project\_number |  |
-| service\_account\_display\_name | The display name of the default service account |
-| service\_account\_email | The email of the default service account |
-| service\_account\_id | The id of the default service account |
-| service\_account\_name | The fully-qualified name of the default service account |
-| service\_account\_unique\_id | The unique id of the default service account |
+| group_email | The email of the GSuite group with group_name |
+| project_bucket_self_link | Project's bucket selfLink |
+| project_bucket_url | Project's bucket url |
+| project_id |  |
+| project_name |  |
+| project_number |  |
+| service_account_display_name | The display name of the default service account |
+| service_account_email | The email of the default service account |
+| service_account_id | The id of the default service account |
+| service_account_name | The fully-qualified name of the default service account |
+| service_account_unique_id | The unique id of the default service account |
```

Comment:
This and similar are a result of the new terraform docs behavior in 0.12 and the result of `make generate_docs`

Reply:
Was using an old version of terraform-docs, updated to current v0.6.0 with terraform 0.12.3 and the docs patch became one line.